core/src/main/java/com/linecorp/armeria/server/HttpResponseSubscriber.java

```diff
@@ -200,6 +219,18 @@ public void onNext(HttpObject o) {
                     newHeaders.remove(HttpHeaderNames.CONTENT_LENGTH);
                 }
 
+                if (config.useServerHeader() &&
+                    !newHeaders.contains(HttpHeaderNames.SERVER)) {
+                    newHeaders.add(HttpHeaderNames.SERVER, serverHeader);
+                }
+
+                if (config.useDateHeader() &&
+                    !newHeaders.contains(HttpHeaderNames.DATE)) {
+                    newHeaders.add(HttpHeaderNames.DATE,
+                                   DateTimeFormatter.RFC_1123_DATE_TIME.format(
+                                           ZonedDateTime.now(ZoneOffset.UTC)));
```

Comment:
question : 
AFAIU, Server need to use `RFC_1123_DATE_TIME` format by `RFC 1123`
But, I'm not sure which time zone should we use.
In this case, I just followed @anuraaga's example from [here](https://github.com/TechEmpower/FrameworkBenchmarks/blob/4a1aad25baa71e52827d6ea4d1f78dd30dfb153f/frameworks/Java/armeria/src/main/java/hello/App.java#L27)

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date
from the above: 
HTTP dates are always expressed in GMT, never in local time.

Reply:
Thank you for education :+1: